small-talk/pages/events/index.js

```diff
@@ -14,7 +14,55 @@ export const Events = ({data}) =>{
     return(
         <div className='bg-slate-800 text-white'>
             <Layout width={windowWidth} renderHeader={true} renderFooter={true}>
-                <h1>Events Page</h1>
+                <h1 className = 'font-bold'>Today&apos;s Events</h1>
+                <ul className = 'divide-y divide-solid'>
+                    <li className = 'list-disc'>
+                        <ul>
+                            <li>2024-5-2</li>
+                            <li>Toy Drive</li>
+                            <li>Toys donated to the hospital will be distributed to patients starting at 12:00pm.</li>
+                        </ul>
+                    </li>
+                    <li className = 'list-disc'>
+                        <ul>
+                            <li>2024-5-2</li>
+                            <li>Meetup with Timmy</li>
+                            <li>Timmy and I will hangout together and play Roblox!!</li>
+                        </ul>
+                    </li>
+                </ul>
+
+                <h1 className = 'font-bold pt-10'>Upcoming Events</h1>
+                <ul className = 'divide-y divide-solid'>
```

Comment:
The lines of words look a little bunched up. Maybe divide by a line + an extra line if possible?